.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish
+
+on:
+  push:
+    branches:
```

Comment:
Let's run only on a tag push (`v*`) and `workflow_dispatch` (manual). See, for example, https://github.com/anycable/anycable/blob/master/.github/workflows/push-stable.yml